ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -224,16 +227,26 @@ export const ApplicationSummary = (props: {
                                     {app.spec.syncPolicy.automated.selfHeal && (
                                         <button className='argo-button argo-button--base' onClick={() => setAutoSync(
                                             ctx, 'Disable Self Heal?', 'Are you sure you want to disable automated self healing?',
-                                            app.spec.syncPolicy.automated.prune, false)
+                                            app.spec.syncPolicy.automated.prune, false, app.spec.syncPolicy.automated.maintenanceWindows)
                                         }>Disable</button>
                                     ) || (
                                         <button className='argo-button argo-button--base' onClick={() => setAutoSync(
                                             ctx, 'Enable Self Heal?', 'Are you sure you want to enable automated self healing?',
-                                            app.spec.syncPolicy.automated.prune, true)
+                                            app.spec.syncPolicy.automated.prune, true, app.spec.syncPolicy.automated.maintenanceWindows)
                                         }>Enable</button>
                                     )}
                                 </div>
                             </div>
+                            {app.spec.syncPolicy.automated.maintenanceWindows && (
```

Comment:
any chance of a screenshot attached to the PR?

Reply:
![Screenshot 2019-08-22 at 08 47 25](https://user-images.githubusercontent.com/12121539/63496926-b0c25e00-c4ba-11e9-843f-ce40a74e5fdc.png)